distribution/docs/src/main/resources/content/_integrating/_endpoints/wps-endpoint.adoc

```diff
@@ -41,6 +24,22 @@ The operations GetStatus and GetResult are used in conjunction with asynchronous
 For brevity the examples below use GET Key-value pair requests but POST is also supported.
 See the OGC WPS 2.0 Interface Standard for more details.
 
+.WPS Endpoint Operations
+[cols="2,1m,2,5m", options="header"]
+|===
+
+|Operation
+|HTTP Request Type
+|Details
+|Example URL
+
+|
+|{url}
+|
+|
```

Comment:
✏️ The `{url}` is in the wrong column. Should be in the last one.

Reply:
fixed